doc/.sphinx/.markdownlint/doc-lint.sh

```diff
@@ -20,8 +20,11 @@ mdl .tmp/doc -sdoc/.sphinx/.markdownlint/style.rb -udoc/.sphinx/.markdownlint/ru
 
 ## Postprocessing
 
-filtered_errors="$(grep -vxFf doc/.sphinx/.markdownlint/exceptions.txt .tmp/errors.txt)"
-if [ "$(echo "$filtered_errors" | wc -l)" = "2" ]; then
+sed -i '/^$/,$d' .tmp/errors.txt
```

Comment:
To do line deletion, I'm used to seeing the selector regex (`/^$/`) followed by the delete action `d` which would look like `sed -i '/^$/ d' .tmp/errors.txt`.

But your `sed` filter works and I'd like your help to understand it and learn new `sed` tricks! What are the `,` and `$` before the `d` for?

Reply:
He he, I learned all that this morning ... :smile: 
`sed -i '/^$/ d' .tmp/errors.txt` would probably only delete the empty line, right? But we want to delete anything AFTER the empty line (this is what has been added in the new mdl release).

The `,` defines an address range, and `$` is the end of the file. So `/^$/,$d` deletes (`d`) everything from the `/^$/` regex (the empty line) until the end of the file (`$`).

Reply:
Many thanks, this make a lot of sense once you spelled it out for me :)

Reply:
`sed -i '/^$/ d' .tmp/errors.txt` would have deletes all the empty lines. A selector doesn't need an end of range to work but in this case, it did need one because it had to delete the empty line and all that followed, hence your end of range `,$` :)